### ISSUE TITLE
[11.0] Frustating behavior when edit views

### ISSUE THREAD

**Issue by DR4RCON** [odoo/odoo#24712]
Impacted versions: 11

Steps to reproduce: Install current head. Install some basic modules. Activate developer mode. Go to settings -> views -> edit any -> Click with mouse into the editor at position you would like to edit.

Current behavior: The form jumps to top or buttom.

Expected behavior: The form does not jump to top or button when click into the editor, but stands still.

![peek 2018-05-14 15-50](https://user-images.githubusercontent.com/30311920/40001570-c7f83dc8-578e-11e8-9b32-58453adfc4fb.gif)

**Comment by pedrobaeza** [comment]
That's not a fresh install, as you have a community theme and maybe some other community modules, so you have to try with only core modules (or in runbot). Meanwhile, I close this.

**Comment by DR4RCON** [comment]
Pedro I have tried with fresh install and only demo data and modul website and it is same. Dont recognize the theme of that gif. Here you have a gif from fresh install.

![peek 2018-05-14 16-18](https://user-images.githubusercontent.com/30311920/40002993-8638de34-5792-11e8-8d39-f1be33240ef2.gif)

**Comment by Yenthe666** [comment]
@pedrobaeza that is in fact default behaviour in Odoo. Nothing to do with custom code.

**Comment by pedrobaeza** [comment]
Then indeed I think this can be consider a bug, or at least an annoyance behavior.

@qsm-odoo is that you the one for checking this?

**Comment by fmdl** [comment]
I confirm this bug in V10 and V11.

**Comment by qsm-odoo** [comment]
Mmh, strange. This bug was fixed weeks (months?) ago by @ged-odoo . I tested and it is indeed fixed in enterprise version but still appears in community version (which does not make sense to me as it was a general fix for both versions). Anyways, @ged-odoo 's team is the one to handle this kind of bug :wink:

**Comment by pedrobaeza** [comment]
Thanks, @qsm-odoo for the insight and ping!

**Comment by vcloud9ray** [comment]
I confirm this is fixed in MASTER Enterprise on Runbot - tested at http://329345-master-27e9dc.runbot16.odoo.com/web/login - but with v10 Enterprise on Runbot - tested at http://329558-10-0-ee00d4.runbot16.odoo.com/web/login - clicking in the view scrolls the entire view so the link you clicked on is the very last line on the screen.

Clicking in the view shouldn't have any effect on scrolling, IMHO.

**Comment by ged-odoo** [comment]
the issue seems to be an issue fixed with an update of the ace library. In theory, it is fixed in saas-11.1 (or 11.2? or 11.3).  A library update is a breaking change, so not really a good idea to put it in 10.0/11.0 (even though it should work I guess)

**Comment by vcloud9ray** [comment]
I agree a library update is breaking the 'stable' release policy - thanks for the update @gde-odoo !

**Comment by DR4RCON** [comment]
What does this mean? Is this fixed?

**Comment by Yenthe666** [comment]
@DR4RCON it means it'l be fixed in the master (upcoming V12) version.
